### PR TITLE
Add scrollEnabled prop to Android ScrollView

### PR DESCRIPTION
**Pull request by brentvatne**
Perhaps there is a better way to do this, curious to hear it!

- If momentum scroll is active when `scrollEnabled` is toggled, the momentum scroll continues and the onMomentumScrollEnd event fires, which is the same as on iOS.

### CODE REVIEW

**Comment by dmmiller** [comment]
Didn't I see another scrollEnabled pr recently?  https://github.com/facebook/react-native/pull/5330

Can we look at getting these two in sync?

Also, can you update the .js file to indicate it is no longer an ios only property

**Comment by brentvatne** [comment]
Hi @dmmiller! Thanks for pointing that out, I didn't notice that PR. It looks like it hasn't been active for a couple weeks so I pulled in the changes that he did to horizontal scroll view as well and made sure this PR addressed yours and @mkonicek's feedback.

**Comment by dmmiller** [comment]
Thanks for cleaning up

@facebook-github-bot shipit

File: ReactAndroid/src/main/java/com/facebook/react/views/scroll/ReactHorizontalScrollViewManager.java [hunk 1]
```diff
@@ -39,17 +39,21 @@ public ReactHorizontalScrollView createViewInstance(ThemedReactContext context)
     return new ReactHorizontalScrollView(context);
   }
 
-  @ReactProp(name = ReactClippingViewGroupHelper.PROP_REMOVE_CLIPPED_SUBVIEWS)
-  public void setRemoveClippedSubviews(ReactHorizontalScrollView view, boolean removeClippedSubviews) {
-    view.setRemoveClippedSubviews(removeClippedSubviews);
+  @ReactProp(name = "scrollEnabled", defaultBoolean = true)
+  public void setScrollEnabled(ReactScrollView view, boolean value) {
```
Comment (dmmiller) [inline ReactAndroid/src/main/java/com/facebook/react/views/scroll/ReactHorizontalScrollViewManager.java]: copy/paste error.  This should be ReactHorizontalScrollView

Comment (brentvatne) [inline ReactAndroid/src/main/java/com/facebook/react/views/scroll/ReactHorizontalScrollViewManager.java]: yikes

Comment (brentvatne) [inline ReactAndroid/src/main/java/com/facebook/react/views/scroll/ReactHorizontalScrollViewManager.java]: fixed, sorry about that!

**Comment by brentvatne** [comment]
@facebook-github-bot shipit

**Pull request closed without merging**